Makefile

```diff
@@ -28,8 +29,20 @@ test:
 	go get -t ./...
 	go get github.com/golang/lint/golint
 	go test -v -race ./...
-	go vet ./...
-	golint ./...
+	go vet github.com/taskcluster/taskcluster-worker/codegen/...
+	go vet github.com/taskcluster/taskcluster-worker/commands/...
+	go vet github.com/taskcluster/taskcluster-worker/config/...
+	go vet github.com/taskcluster/taskcluster-worker/engines/...
+	go vet github.com/taskcluster/taskcluster-worker/plugins/...
+	go vet github.com/taskcluster/taskcluster-worker/runtime/...
+	go vet github.com/taskcluster/taskcluster-worker/worker/...
+	golint github.com/taskcluster/taskcluster-worker/codegen/...
+	golint github.com/taskcluster/taskcluster-worker/commands/...
+	golint github.com/taskcluster/taskcluster-worker/config/...
+	golint github.com/taskcluster/taskcluster-worker/engines/...
+	golint github.com/taskcluster/taskcluster-worker/plugins/...
+	golint github.com/taskcluster/taskcluster-worker/runtime/...
+	golint github.com/taskcluster/taskcluster-worker/worker/...
```

Comment:
Note, this seems to be a limitation in the language (not related to the vendor tool). If you look at `go help packages` you'll see that `./...` would be expanded to match all packages under the current package, but strangely this also seems to include vendored packages, so `./...` or even `github.com/taskcluster/taskcluster-worker/...` are no good to us. For this reason I've had to list the packages explicitly, I haven't found a way around this.